pkg/reconciler/route/route.go

```diff
@@ -113,11 +114,15 @@ func (c *Reconciler) ReconcileKind(ctx context.Context, r *v1.Route) pkgreconcil
 		return err
 	}
 
-	logger.Info("Updating targeted revisions.")
-	// In all cases we will add annotations to the referred targets.  This is so that when they become
-	// routable we can know (through a listener) and attempt traffic configuration again.
-	if err := c.reconcileTargetRevisions(ctx, traffic, r); err != nil {
-		return err
+	if cfgmap.FromContextOrDefaults(ctx).Features.ResponsiveRevisionGC == cfgmap.Disabled {
+		logger.Info("ResponsiveGC enabled. Skipping LastPinned updates.")
+	} else {
```

Comment:
We hardly want this message on every route reconcile? I'd just leave this `if` branch out

Reply:
It seems just as useful (or not) as `logger.Info("Updating targeted revisions.")` on the other branch on every route reconcile?

Reply:
Sure the other can go too :)